winpr/libwinpr/pool/pool.c

```diff
@@ -1,3 +1,8 @@
+#ifdef __MINGW32__
+#define _TP_CALLBACK_INSTANCE S_TP_CALLBACK_INSTANCE
+#define _TP_POOL S_TP_POOL
+#define _TP_WORK S_TP_WORK
+#endif
 /**
```

Comment:
Oops, it wasn't meant to be upstreamed like this. This shouldn't stop review, but the lines can't stay at the top.